OWNERS

```diff
@@ -0,0 +1,3 @@
+@coreos-tools-team
```

Comment:
Does this work?  I thought we needed at least things under `approvers:`.

In e.g. https://github.com/coreos/rpm-ostree/pull/1916/files I found it useful to add a link to the Prow docs on the file format.

Reply:
I was using the Github format. Go figure.